pkgs/tools/graphics/svgcleaner/default.nix

```diff
@@ -1,22 +1,27 @@
-{ lib, fetchFromGitHub, rustPlatform }:
+{ lib, rustPlatform, fetchFromGitHub }:
 
 rustPlatform.buildRustPackage rec {
   pname = "svgcleaner";
-  version = "0.9.2";
+  version = "0.9.5";
 
   src = fetchFromGitHub {
     owner = "RazrFalcon";
     repo = "svgcleaner";
     rev = "v${version}";
-    sha256 = "1jpnqsln37kkxz98vj7gly3c2170v6zamd876nc9nfl9vns41s0f";
+    sha256 = "sha256-nc+lKL6CJZid0WidcBwILhn81VgmmFrutfKT5UffdHA=";
   };
 
-  cargoSha256 = "172kdnd11xb2qkklqdkdcwi3g55k0d67p8g8qj7iq34bsnfb5bnr";
+  cargoLock.lockFile = ./Cargo.lock;
+
+  postPatch = ''
```

Comment:
Does this work without being cargoPatches?

Reply:
yes, `cargoLock` option doesn't check of do anything to `Cargo.lock`, unlike `cargoSha256` and the likes which verifies that `Cargo.lock` is the same